scripts/get_tlds.py

```diff
@@ -1,8 +1,22 @@
+import re
 import requests
 
-tdls = requests.get("https://data.iana.org/TLD/tlds-alpha-by-domain.txt")
-final_string = "|".join(tdls.text.split("\n")[1:-1])
+tlds = requests.get("https://data.iana.org/TLD/tlds-alpha-by-domain.txt")
+final_string = "|".join(tlds.text.split("\n")[1:-1])
 
-file = open("tld_list.txt", "w")
-file.write(final_string)
-file.close()
+
+with open("pywhat/Data/regex.json", "r") as file:
+    database = file.read()
+
+database = re.sub(
+    # what you want to replace
+    r'("Name": "Uniform Resource Locator \(URL\)",\s*"Regex": ")[^"]+(")',
+
+    # what you want to replace it with - need to be double slashes
+    r"\1(?i)^(?:(?:https?|ftp):\/\/(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\\\\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)(?::[0-9]{1,5})?)([a-z0-9-_\/]?\\\\.?)*$|^(?:(?:https?|ftp):\/\/)?(?:\\\\S+:\\\\S+@)?(?:[a-z0-9-_]?\\\\.?)+?(?:[a-z0-9-_][a-z0-9-_]{0,62})\\\\.(?:" + final_string + r")(?::\\\\d{2,5})?(?:[\/?#]\\\\S*)?$\2",
+
+    database
```

Comment:
This is very hard-core I would have just:
* Turned the file into JSON
* Find the URL regex's json
* Update the URL regex in the python dictionary
* `json.dump()` the dictionary back into the file

I don't really trust regex well enough to do this, but if it works it works!

Reply:
Occams Razor:

> a scientific and philosophical rule that entities should not be multiplied unnecessarily which is interpreted as requiring that the simplest of competing theories be preferred to the more complex or that explanations of unknown phenomena be sought first in terms of known quantities.

AKA:

> The simplest solutions are always the best

And:

> The golden hammer, if you are familiar with a hammer then everything looks like a nail

That doesn't mean everything is best solved with a hammer, you're using regex here because you're most familiar with it I believe. Sometimes we have to step back and look at better tooling 😄 
https://en.wikipedia.org/wiki/Law_of_the_instrument

Reply:
> This is very hard-core I would have just:
> 
> * Turned the file into JSON
> 
> * Find the URL regex's json
> 
> * Update the URL regex in the python dictionary
> 
> * `json.dump()` the dictionary back into the file
> 
> 
> 
> I don't really trust regex well enough to do this, but if it works it works! 

I did actually try that at first! The issue was that it outputted to the file everything "together". Like it wasn't every value in each line like we have it now.